app/models/ability.rb

```diff
@@ -0,0 +1,51 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # can :read, :all
+    # return unless user
+
+    # # can %i[read create destroy], [Post, Comment, Like]
+    # can :manage, Post, author_id: user.id
+    #   can :manage, Comment, author_id: user.id
+    #   can :read, :all
+
+    # return unless user.role == 'admin'
+
+    # can :manage, :all
```

Comment:
- [ ] [OPTIONAL] Kindly, remove the commented code from your codebase.

Reply:
@RolandM99 Thank you for the approval, I will remove them